lib/matplotlib/tests/test_sphinxext.py

```diff
@@ -12,11 +12,13 @@
 
 
 def test_tinypages(tmpdir):
-    html_dir = pjoin(str(tmpdir), 'html')
-    doctree_dir = pjoin(str(tmpdir), 'doctrees')
+    tmpdir = Path(tmpdir)
```

Comment:
```suggestion
    tmp_path = Path(tmpdir)
```
would be even closer :).

Reply:
Good point.